### DISCUSSION TITLE
Mission Equality Check failure preventing players from joining

### DISCUSSION THREAD

**Discussion by bananaman6213** [FakeFishGames/Barotrauma#13803] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

Players cannot join a lobby while missions are selected at an outpost, as described in #11469. This issue was marked as a duplicate of another mission equality check issue but i believe that the two were unrelated.

### Reproduction steps

1. Host lobby
2. In an outpost, select missions on the map screen without actually starting a round
3. Have someone else join
4. They should be kicked due to mission equality check failure
5. Deselect the missions
6. Watch as that second player can now join without issue

### Bug prevalence

Happens every now and then

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.3.0.4

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly. Fix coming in the next update. Closing discussion.
